platform/jobs/RetirementJobEdxTriggers.groovy

```diff
@@ -0,0 +1,92 @@
+/* UserRetirementJobEdxTriggers.groovy
+ *
+ * Defines jobs which trigger downstream user retirement jobs for each edX
+ * environment.
+ */
+
+package platform
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_TEAM_SECURITY
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+List jobConfigs = [
+    [
+        environmentDeployment: 'prod-edx'
+    ],
+    [
+        environmentDeployment: 'stage-edx'
+    ],
+    [
+        environmentDeployment: 'loadtest-edx'
+    ],
+    [
+        environmentDeployment: 'prod-edge'
+    ]
+]
+
+
+jobConfigs.each { jobConfig ->
+
+    // ########### user-retirement-trigger-<environment> ###########
+    // This defines the job which triggers the collector job for a given environment.
+    job("user-retirement-trigger-${jobConfig.environmentDeployment}") {
+        description("Scheduled trigger of the user-retirement-collector job for the ${jobConfig.environmentDeployment} environment")
+
+        // Only a subset of edx employees should be allowed to control this job,
+        // but customer support can read and discover.
+        authorization {
+            blocksInheritance(true)
+            List membersWithFullControl = ['edx/platform-team']
```

Comment:
I just realized that this, and the configuration of the children jobs, will lock testeng out from even discovering that these jobs exist (let alone managing them). We will need to be able to.